# Request
string division
string object_name
string object_size

# Response
---
string object_namespace